src/py42/util.py

```diff
@@ -72,3 +72,30 @@ def format_dict(dict_, label=None):
     if label:
         return u"{} {}".format(label, indented_dict)
     return indented_dict
+
+
+def bool_to_str(value):
+    if isinstance(value, bool):
+        return "true" if value else "false"
+    else:
+        raise KeyError("Value must be True or False")
+
+
+def str_to_bool(value):
+    if value == "true":
+        return True
+    if value == "false":
+        return False
+    else:
+        return value
```

Comment:
Why not just return `None` here?

Reply:
or raise `KeyError`

Reply:
I think I was letting the `else` capture if a boolean was passed that we'd just let it through, but it might be better to raise since this func is making an explicit conversion assumption and we wouldn't want to hide a bug by it being used incorrectly.